mender-deb-package

```diff
@@ -50,6 +50,24 @@ checkout_repo() {
   git checkout $VERSION
 }
 
+install_go() {
+ local GOLANG_VERSION=1.14.7
+ local golang_version_set
+
+ golang_version_set=$(sed -n '/[^ ]*GOLANG_VERSION:[ "0-9]/{s/.*GOLANG_VERSION:[ ]*\(["0-9\.]*\).*/\1/p;q}' .gitlab-ci.yml)
+ golang_version_set=${golang_version_set//\"/}
+ GOLANG_VERSION=${golang_version_set:-${GOLANG_VERSION}}
+
+ wget -q https://dl.google.com/go/go$GOLANG_VERSION.linux-amd64.tar.gz \
```

Comment:
Will this work? In the other PRs we are setting `GOLANG_VERSION` to `1.17` (missing bugfix number).

Reply:
It works :+1: